src/pages/privacy-policy.js

```diff
@@ -2,13 +2,21 @@ import React from 'react';
 
 import s from '../components/layout/ProjectLandingPage/styles.module.scss';
 
-const PrivacyPage = () => (
-  <main className={s.main}>
-    <div className={s.innerContent}>
-      <h3>Cookie List from Modus Labs</h3>
-      <article><div id="ot-sdk-cookie-policy" /></article>
-    </div>
-  </main>
-);
+const PrivacyPage = () => {
+  React.useEffect(() => {
+    setTimeout(() => {
+      window.Optanon.Close();
+    }, 3000);
```

Comment:
This code is not tested, but adding timeouts on effects can lead to memory leaks, so something along the following should be set to prevent issues.

```suggestion
    const timer = setTimeout(() => {
      window.Optanon.Close();
    }, 3000);
    return clearTimeout(timer);
```

Reply:
added